data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/range_action/PstRange.java

```diff
@@ -51,8 +51,8 @@ public void apply(Network network, double setpoint) {
         if (phaseTapChanger == null) {
             throw new FaraoException(String.format("Transformer %s is not a PST, tap could not be changed", networkElement.getId()));
         }
-        if (phaseTapChanger.getHighTapPosition() >= setpoint && phaseTapChanger.getLowTapPosition() <= setpoint) {
-            phaseTapChanger.setTapPosition((int) setpoint);
+        if (phaseTapChanger.getHighTapPosition() - phaseTapChanger.getLowTapPosition() >= setpoint) {
```

Comment:
Hmmm

Reply:
yes ?

Reply:
Are you sure that setpoint=33 is possible? :aerial_tramway:

Reply:
not really, no... but now I guess... it will work... I hope :cactus:

Reply:
:crossed_fingers: :four_leaf_clover: